src/main/groovy/de/dkfz/roddy/execution/jobs/BatchEuphoriaJobManager.groovy

```diff
@@ -72,8 +69,7 @@ abstract class BatchEuphoriaJobManager<C extends Command> {
         this.queryOnlyStartedJobs = parms.trackOnlyStartedJobs
         this.userIDForQueries = parms.userIdForJobQueries
         if (!userIDForQueries && isTrackingOfUserJobsEnabled) {
-            logger.warning("Silently falling back to default job tracking behaviour. The user name was not set properly and the system cannot track the users jobs.")
-            isTrackingOfUserJobsEnabled = false
+            throw new BEException("The user name was not set properly and the system cannot track the users jobs.")
```

Comment:
At least for Roddy, this could come up as an issue, please open an issue describing this change.